calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/IcalendarReader.java

```diff
@@ -53,7 +53,7 @@
  */
 @Slf4j
 public class IcalendarReader extends Reader {
-	public static final long MINUTES_IN_DAY = TimeUnit.DAYS.toMinutes(1);
+	public static final long MINUTES_IN_DAY = TimeUnit.DAYS.toMinutes(1) - 5;
```

Comment:
this line is unrelated but it addresses an issue in the ui where events are on 5 minute intervals